readme.md

```diff
@@ -19,7 +18,19 @@
 
 ## Inspect the Dashboard Performance in the Dashboard Designer
 
+1. Download the [Dashboard Diagnostic Tool](https://github.com/DevExpress-Examples/bi-dashboard-diagnosic-tool). 
+
+2. Reference `DiagnosticTool.dll` and install the [Microsoft.Diagnostics.Tracing.TraceEvent](https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing.TraceEvent/) package in your dashboard project. 
+
+3. Create a custom button and insert it to the Ribbon.
+
+4. Create a `DiagnosticController` object. 
+
+5. Call the conroller's Start() and Stop() methods on button's click to run and finish the Dashboard Diagnostic Tool's session.

```

Comment:
Note that "conroller's" is mispelled.
Also, what do you mean by "on button's click"? For example, do you mean:

`5. Call the controller's Start() and Stop() methods on a button click to run and finish the Dashboard Diagnostic Tool's session.`

or

`5. Call the controller's Start() and Stop() methods on each button click to run and finish the Dashboard Diagnostic Tool's session.`

Reply:
```suggestion
5. Call the controller's Start() and Stop() methods on a button click to run and finish the Dashboard Diagnostic Tool's session.

```